src/core/log.h

```diff
@@ -22,4 +22,7 @@ void log_info(const char *msg, const char *param_str, int param_int);
  */
 void log_error(const char *msg, const char *param_str, int param_int);
 
+void log_window_init();
```

Comment:
For function that don't accept any arguments, you need to declare them with void in the argument list, otherwise they will accept any arguments.